### PR TITLE
Making debug message formatting disabled when debug is disabled

### LINKED ISSUE #446: TCK performance
Thank you for providing the TCK test suite. It's very useful!

I'm having troubles with its performance, though. I've profiled it with yourkit and it seems that most of the time is spent formatting strings for debugging even if debugging is disabled:

screen shot 2018-12-29 at 11 49 18 pm

### PR DESCRIPTION
**Pull request by viktorklang**
Fixes #446

### CODE REVIEW

**Comment by viktorklang** [comment]
@reactive-streams/contributors Pretty non-controversial PR, so I'll merge this shortly.

File: tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java [hunk 1]
```diff
@@ -504,7 +504,8 @@ public TestSetup apply(Long aLong) throws Throwable {
 
           @Override
           public void onSubscribe(final Subscription subscription) {
-            env.debug(String.format("whiteboxSubscriber::onSubscribe(%s)", subscription));
+            if (env.debugEnabled())
```
Comment (JakeWharton) [inline tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java]: Needs curly braces to adhere to codebase style.

Comment (viktorklang) [reply tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java]: @JakeWharton Good point—addressed!

**Comment by viktorklang** [comment]
@reactive-streams/contributors Merging.

**Pull request merged**